src/main/scala/com/gu/fezziwig/CirceScroogeMacros.scala

```diff
@@ -238,17 +238,17 @@ private class CirceScroogeMacrosImpl(val c: blackbox.Context) {
     q"""{
       new _root_.io.circe.Decoder[$A] {
         def apply(c: _root_.io.circe.HCursor): _root_.io.circe.Decoder.Result[$A] = {
-          val result = c.fields.getOrElse(Nil).headOption.flatMap {
+          val maybeResult = c.fields.getOrElse(Vector.empty).headOption.flatMap {
```

Comment:
`c.fields` returns a `Option[Vector]` (used to be a List). Changing it from Nil makes no difference as far as I know, as `headOption` is common to both, but it feels more correct.